cmd/vclusterctl/cmd/platform/add/vcluster.go

```diff
@@ -29,7 +29,12 @@ Adds a vCluster to the vCluster platform.
 
 Example:
 vcluster platform add vcluster my-vcluster --namespace vcluster-my-vcluster --project my-project --import-name my-vcluster
+
+Add all vClusters in the host cluster:
+vcluster platform add vcluster ignored --project my-project --all
```

Comment:
Instead of forcing them to pass an ignored vcluster name, set [Args](https://github.com/loft-sh/vcluster/blob/cf9a1a48a2b38594df2c4139bbb1d62f23f9bf69/cmd/vclusterctl/cmd/platform/add/vcluster.go#L39) to `RangeArgs(0, 1)` then return an error if args is 0 and `--all` is false.

Reply:
Yes, please change that

Reply:
good point!